bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/shape-draw-listener/component.jsx

```diff
@@ -298,7 +298,7 @@ export default class ShapeDrawListener extends Component {
 
   render() {
     const { tool } = this.props.drawSettings;
-    const baseName = Meteor.settings.public.app.basename;
+    const baseName = Meteor.settings.public.app.cdn || Meteor.settings.public.app.basename;
```

Comment:
The other ones use cnd + basename, but this one is cdn || basename. Is that intentional?

Reply:
Same as previous.